website/docs/guides/container_service_extension_4_0.html.markdown

```diff
@@ -0,0 +1,411 @@
+- `solutions_routed_network_ip_pool_end_address`: The [Routed network][routed_network] that will be created in the Solutions Organization will have a pool of usable IPs, this field
+  defines the end usable IP.
+- `solutions_snat_external_ip`: This is used to create a SNAT rule on the Solutions Edge Gateway to provide Internet connectivity to the CSE Server. The external IP should be one available IP of the Solutions
+  [Provider Gateway][provider_gateway].
+- `solutions_snat_internal_network_cidr`: This is used to create a SNAT rule on the Solutions Edge Gateway to provide Internet connectivity to the CSE Server. The subnet should correspond to the Solutions
+  Organization [Routed network][routed_network].
+- `solutions_routed_network_dns`: DNS Server for the Solutions Organization [Routed network][routed_network]. It can be left blank if it's not needed.
+- `tenant_routed_network_gateway_ip`: The gateway IP of the [Routed network][routed_network] that will be created in the Tenant Organization.
+- `tenant_routed_network_prefix_length`: The prefix length of the [Routed network][routed_network] that will be created in the Tenant Organization.
+- `tenant_routed_network_ip_pool_start_address`: The [Routed network][routed_network] that will be created in the Tenant Organization will have a pool of usable IPs, this field
+  defines the first usable IP.
+- `tenant_routed_network_ip_pool_end_address`: The [Routed network][routed_network] that will be created in the Tenant Organization will have a pool of usable IPs, this field
+  defines the end usable IP.
+- `tenant_snat_external_ip`: This is used to create a SNAT rule on the Tenant Edge Gateway to provide Internet connectivity to the clusters. The external IP should be one available IP of the Tenant
+  [Provider Gateway][provider_gateway].
+- `tenant_snat_internal_network_cidr`: This is used to create a SNAT rule on the Tenant Edge Gateway to provide Internet connectivity to the clusters. The subnet should correspond to the Tenant
+  Organization [Routed network][routed_network].
+- `tenant_routed_network_dns`: DNS Server for the Tenant Organization [Routed network][routed_network]. It can be left blank if it's not needed.
+
+If you wish to have a different networking setup, please modify the [proposed configuration][step2].
+
+### CSE Server
+
+The final set of resources created by the [proposed configuration][step2] correspond to the CSE Server vApp.
+The generated VM makes use of the uploaded CSE OVA and some required guest properties.
+
+In order to do so, the [configuration][step2] asks for the following variables that you can customise in `terraform.tfvars`:
+
+- `vcdkeconfig_template_filepath`: This references a local file that corresponds with the `VCDKEConfig` [RDE][rde] contents specified as a JSON template.
+  You can find this template [here](https://github.com/vmware/terraform-provider-vcd/tree/main/examples/container-service-extension-4.0/entities/vcdkeconfig-template.json).
```

Comment:
This is a bit overwhelming for a fresh user :) Unfortunately, I don't have an idea how to simplify it. Maybe this (it may not be correct anymore though):

```suggestion
- `vcdkeconfig_template_filepath`: references a local file that defines the `VCDKEConfig` schema ([RDE][rde] contents) specified as a JSON template.
  You can find this template [here](https://github.com/vmware/terraform-provider-vcd/tree/main/examples/container-service-extension-4.0/entities/vcdkeconfig-template.json).
```

Maybe we should also add "It must match CSE version used." or similar sentence.

CC: @Didainius @adezxc @dataclouder (for ideas)

Reply:
I changed it with slightly different words, as the JSON is not really the schema, but the RDE contents.
But I hope it's improved now.